gobblin-utility/src/main/java/org/apache/gobblin/util/filesystem/DataFileVersionStrategy.java

```diff
@@ -65,12 +65,13 @@
   }
 
   String DATA_FILE_VERSION_STRATEGY_KEY = "org.apache.gobblin.dataFileVersionStrategy";
+  String DEFAULT_DATA_FILE_VERSION_STAREGY = "modtime";
```

Comment:
can you fix the spelling of the key name?

Reply:
Fixed.